pkg/reconciler/apis/apibinding/conflict_checker.go

```diff
@@ -53,33 +53,13 @@ func (ncc *conflictChecker) getBoundCRDs(apiBindingToExclude *apisv1alpha1.APIBi
 		if apiBinding.Name == apiBindingToExclude.Name {
 			continue
 		}
-
-		apiExportClusterName, err := getAPIExportClusterName(apiBinding)
-		if err != nil {
-			return err
-		}
-
-		apiExport, err := ncc.getAPIExport(apiExportClusterName, apiBinding.Spec.Reference.Workspace.ExportName)
-		if err != nil {
-			return err
-		}
-
 		boundSchemaUIDs := sets.NewString()
 		for _, boundResource := range apiBinding.Status.BoundResources {
 			boundSchemaUIDs.Insert(boundResource.Schema.UID)
 		}
 
-		for _, schemaName := range apiExport.Spec.LatestResourceSchemas {
-			schema, err := ncc.getAPIResourceSchema(apiExportClusterName, schemaName)
-			if err != nil {
-				return err
-			}
-
-			if !boundSchemaUIDs.Has(string(schema.UID)) {
-				continue
-			}
-
-			crd, err := ncc.getCRD(ShadowWorkspaceName, string(schema.UID))
+		for _, bs := range apiBinding.Status.BoundResources {
```

Comment:
mhm, I think the apibinding controller sets the `BoundResources` after the checker says it is okay. Thus relying only on `apiBinding.Status.BoundResources` seems wrong, no?

Reply:
But this is about "the other" bound resources, not of the current object.

Reply:
right, maybe it is actually even better. In case of a conflict (concurrent APIExports) we will be able at least bind one CRD. I think previously we would reject both.

Reply:
yes, chance for something to break is N times higher